webpack/webpack.common.js

```diff
@@ -17,7 +17,7 @@ function getEntries(pattern) {
   });
 
   entries.svgSprite = path.resolve(webpackDir, 'svgSprite.js');
-  entries.css = path.resolve(webpackDir, 'css.js');
+  entries.css = path.resolve(webpackDir, '../components/style.scss');
```

Comment:
do we want to remove the 'css.js' file then?

Reply:
Good point. Removed css.js since it's no longer needed.